packages/clay-date-picker/src/index.tsx

```diff
@@ -17,13 +17,21 @@ import React from 'react';
 import DateNavigation from './DateNavigation';
 import DayNumber from './DayNumber';
 import DaysTable from './DaysTable';
-import {isValid, parseDate, setDate} from './Helpers';
+import {
+	formatDate,
+	fromRangeToString,
+	fromStringToRange,
+	isDateRangeWithinYears,
+	isValid,
+	parseDate,
+	setDate,
+} from './Helpers';
```

Comment:
Seems like several of these functions are used _only_ in this file (eg. `fromRangeToString`, `fromStringToRange`, `isDateRangeWithinYears`) so by putting them there, you are just obliging the reader to jump into another file to see what they do. My recommendation would be to put functions close to where they are needed (improving "locality" is worth doing for many other kinds of abstractions too, not just functions).

Reply:
Makes sense. Initially I didn't move these functions to Helpers file but after the first review I moved it for reducing the LOC in the same file. :)

Reply:
So, just makes sense moving the Helpers files if a function is used in more than more file